dnf5daemon-server/dbus/interfaces/org.rpm.dnf.v0.rpm.Rpm.xml

```diff
@@ -434,11 +434,11 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
     <!--
         write_to_fd_finished:
-        @success: true if the tranfer finished successfully
-        @transfer_id: id of the tranfer was finished and its file descriptor closed
+        @success: true if the transfer finished successfully
+        @transfer_id: id of the transfer was finished and its file descriptor closed
         @error_msg: error message if the transfer was not successful.
 
-        Processing of the item has started.
+        Notify the user that writing to the file descriptor with given transfer_id has finished.
```

Comment:
Here `has finished` is used. A few lines above, `was finished` is used. I am not an English specialist. I'm just considering whether this inconsistency is correct.